recipes/numcpp/all/conanfile.py

```diff
@@ -57,5 +71,13 @@ def package_id(self):
         self.info.header_only()
         
     def package_info(self):
+        if tools.Version(self.version) >= "2.5.0" and self.options.use_boost == False:
+            self.cpp_info.cxxflags.append("-DNUMCPP_NO_USE_BOOST")
```

Comment:
```suggestion
        if self.options.get_safe("use_boost", False):
            self.cpp_info.cxxflags.append("-DNUMCPP_NO_USE_BOOST")
```

Reply:
Suggested change is not same logic.
I will change these ocde like following:

```suggestion
        if self.options.get_safe("use_boost", False) == False:
            self.cpp_info.cxxflags.append("-DNUMCPP_NO_USE_BOOST")
```